src/rdbms/mongoose_rdbms.erl

```diff
@@ -241,6 +244,17 @@ execute_cast(HostType, Name, Parameters) when is_atom(Name), is_list(Parameters)
 execute_request(HostType, Name, Parameters) when is_atom(Name), is_list(Parameters) ->
     sql_request(HostType, {sql_execute, Name, Parameters}).
 
+-spec execute_measured_request(
+    HostType :: server(),
+    Name :: atom(),
+    Parameters :: [term()],
+    MeasureFun :: fun((fun(() -> T)) -> T)
+) -> request_id().
+execute_measured_request(HostType, Name, Parameters, MeasureFun)
```

Comment:
or you can add sql_dirty_request function and just have execute inside :D 
But I actually like that it is a separate function, good for later tracing